### DISCUSSION TITLE
New Device request: LG air purifier

### DISCUSSION THREAD

**Discussion by northpolebass** [nVuln/homebridge-lg-thinq#161] [Ideas]
It would be great if you could add the new LG Model FS061P*S* to the supported devices. Thanks!

PS. When setting up the configuration, you ask for the account ID, which is different than the account email address. For me, only the email address worked, after wasting a lot of time trying to get the account ID (in my case, my phone number) to work. You might want to mention using the email address in the set up.

Also, I also could only get my non-US account to work after logging out on my phone and again back in. Again, you might want to mention that in your set up instructions.

**Comment by nVuln** [comment]
thanks for your suggestion
<strike>currently this plugin supported Air Purifier, if your device not appear, please try turn on homebridge debug mode and get some debug logging (open new issue for easier tracking)</strike>

I just checked your model number, it's aerotower, new kind of air purifier, I'm still working on this, you can follow it here #145
